packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx

```diff
@@ -118,14 +117,6 @@ const CompareCFDs = observer(() => {
                                         is_demo={is_demo}
                                     />
                                 )}
-                                {/* Renders cTrader data */}
```

Comment:
since this is the temporary removal, it's better to comment these

Reply:
If we comment these, sonar cloud would fail

Reply:
then it's fine